coreneuron/utils/reports/nrnreport.cpp

```diff
@@ -241,7 +241,8 @@ void register_soma_report(NrnThread& nt,
                            config.report_dt, sizemapping, (char*)config.type_str, extramapping,
                            (char*)config.unit);
 
-        /** add extra mapping */
+        records_set_report_max_buffer_size_hint ((char*)config.output_path, config.buffer_size);
```

Comment:
Should this option `report-buffer-size` overwrite the `records_set_report_max_buffer_size_hint` that reads the buffer size from the report.conf file?